lms/static/sass/multicourse/_sysadmin.scss

```diff
@@ -0,0 +1,47 @@
+.mgmt-commands-body #select-command{
+    border: solid 4px black;
+    padding: 20px;
+
+}
+
+.mgmt-commands-body .full-width{
+    width: 100%;
+}
+
+.mgmt-commands-body .bordered-fieldset{
+    border: solid 4px black;
+    padding: 20px;
+    margin-top: 30px;
+}
+
+.mgmt-commands-body .execute-command-button{
+    margin-top: 20px;
+}
+
+.mgmt-commands-body #command-description{
+    margin-top: 10px;
+}
+.mgmt-commands-body select{
+    height: 35px;
+    font-size: 1.1em;
+}
+
+.mgmt-commands-body .error p, .mgmt-commands-body .error h3{
+    color: red;
+
+}
+
+.mgmt-commands-body .stderr p, .mgmt-commands-body .stderr h3{
+    color: maroon;
+}
+.mgmt-commands-body .stdout p, .mgmt-commands-body .stdout h3{
+    color: green;
+}
+
+.mgmt-commands-body .new-section{
+    margin-top: 20px;
+}
+
+.mgmt-commands-body .checkbox-label{
+    display: inline;
+}
```

Comment:
General CSS code style best practice says there should be a space before the opening curly.

ie:
```
.mgmt-commands-body .checkbox-label {
    display: inline;
}
```